feat: readme catalog and gallery (#201)
xmartlabs/flutter-template: catalog/README.md

# Catalog' Flutter template

Catalog is a Flutter package that provides a base implementation for UX/UI components based on the design template of Xmartlabs.
It aims to simplify and expedite the development process for Flutter projects, ensuring a consistent and visually appealing user interface.

> ## Features

Reviewer: Maybe we can add something about the theme here

Reviewer: Also we can explain where each part is, kind of what we have in the template readme with the folder structure.
So it's easier to modify later on

> - Ready-to-use UI components inspired by Xmartlabs design principles.

Reviewer 2: main?
```suggestion
- Ready-to-use main UI components inspired by Xmartlabs design principles.
```

- Customizable components to fit your project's specific requirements.

## Getting started

### Installation

> Add the following line to your `pubspec.yaml` file:

Reviewer 2: ```suggestion
Integrate the package in your project, adding the following line to your `pubspec.yaml` file:
```

```yaml
dependencies flutter:
    catalog:
        path: ../
```

### Contributing
We welcome contributions! If you find a bug or have a feature request, please open an [issue].

> [issue]: https://github.com/xmartlabs/flutter-template

Reviewer 2: ```suggestion
[issue]: https://github.com/xmartlabs/flutter-template/issues
```
